samples/azure-quantum/variational-quantum-eigensolver/VQE-qiskit-hydrogen-ionq-sim.ipynb

```diff
@@ -0,0 +1,564 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "72446518",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# This is a bug that will be addressed in this PR: https://github.com/microsoft/qdk-python/pull/301\n",
+    "ionq_simulator_backend.configuration().max_shots = None"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2ba3e580",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create Quantum Instance\n",
+    "quantum_instance = QuantumInstance(backend=ionq_simulator_backend,\n",
+    "                                   shots=8192)\n",
+    "# Unset qjob config to avoid errors when running job.result()\n",
+    "quantum_instance._qjob_config = {}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e764cfe4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create optimizer with only one iteration\n",
+    "optimizer = SPSA(maxiter=1)\n",
+    "# Create the variational form of the ansatz\n",
+    "var_form = EfficientSU2(qubit_op.num_qubits, entanglement=\"linear\")\n",
+    "# Create a VQE object that runs the algorithm\n",
+    "vqe = VQE(qubit_op, var_form, optimizer=optimizer)\n",
+    "# Set the quantum instance to be able to run only the last iteration\n",
+    "vqe.quantum_instance = quantum_instance"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "abb4e575-c468-4df4-a9af-426829237a16",
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
+   "source": [
+    "The below cell will evaluate the energy at `p0` using the IonQ simulator.\n",
+    "\n",
+    "You have to add the molecular nuclear repulsion energy to the final result to get the ground state of the molecule."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f55a0f10",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "......"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "-1.1328210248074357"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "vqe._energy_evaluation(parameters=p0) + molecule.nuclear_repulsion_energy"
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3 (ipykernel)",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.9.10"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 5
+}
```

Comment:
Can you please add a conclusion cell that wraps the notebook up nicely? It ends a little abruptly. What should I have learned from this notebook? What can I try next? Give the user a "high five" of sorts, too, because they got through this very cool example we should celebrate that!